src/main/java/com/microsoft/graph/Requests/GraphClientFactory.java

```diff
@@ -39,7 +39,6 @@ public static OkHttpClient.Builder create(@Nullable GraphClientOptions options)
     public static Interceptor[] createDefaultGraphInterceptors(@Nullable GraphClientOptions graphClientOptions) {
         List<Interceptor> handlers = new ArrayList<>();
         handlers.add(new GraphTelemetryHandler(graphClientOptions));
-        handlers.add(new CompressionHandler());
```

Comment:
removing this is fine as long as we have a PR to add it back in the Kiota client factory